integration_tests/src/main/python/data_gen.py

```diff
@@ -668,8 +678,10 @@ def gen_scalars_for_sql(data_gen, count, seed=0, force_no_nulls=False):
 boolean_gen = BooleanGen()
 date_gen = DateGen()
 timestamp_gen = TimestampGen()
+decimal_gen = DecimalGen()
```

Comment:
We are going to need more than just one.  We really are going to need a few different sets of decimal_gens. A default set that work with `park.sql.legacy.allowNegativeScaleOfDecimal` disabled, and others that work when it is enabled.

Reply:
Added for default case, positive and negative scale. Thanks!